reporter/gen_report.js

```diff
@@ -32,11 +35,22 @@ Object.keys(originals).forEach(key => {
       escape(atob(originals[key]))));
 });
 
+var fileWriter = new FileWriter(false);
+
+fileWriter.writeFile = function writeFile(file, callback) {
+  if (file.length > 255) {
```

Comment:
I also think most of this logic could be replace w/ a `.slice(0, 255)` approch...

thoughts?